### DISCUSSION TITLE
[Strings & Documentation] Wording, Spelling etc.

### DISCUSSION THREAD

**Discussion by Olf0** [sailfishos-patches/patchmanager#197] [General]
I created a [wiki page "Terms and wording"](https://github.com/sailfishos-patches/patchmanager/wiki/Terms-and-wording) to define consistent spelling and definitions.  I started with the most important ones in the context of Patchmanager.  I think the most debatable definition is "Patch", but for the reasons mentioned there I believe we ought to differentiate.

I plan to slowly integrate this into the documentation and strings, thus I first want to expose this guidance to criticism, deviating suggestions etc.

Additionally I intend to integrate [these punctuation rules](https://github.com/sailfishos-patches/patchmanager/pull/136#discussion_r745053026):
* **!** means: Attention, something went wrong.
* **…** (concluding, i.e., at the end of a phrase or sentence _and_ directly attached, i.e. without a space; note that these are not three dots / full stops, it is a single ellipsis character) means: A new window is going to be opened.   This is most suitable for choices (which would open a new window).
* There are more, which I do not remember spontaneously.  I hope to find a comprehensive write-up in the WWW. 
* The dialogues in MS-Windows and most of KDE and Gnome are adhering to these conventions, so people are used to them (even most not consciously).

**Comment by nephros** [comment]
Thanks, and ~~by the way~~ nice work with the Wiki!

**Reply by Olf0** [reply]
No, not really: Most of it consists of roughly collected items.
There is a lot of structuring, reshuffling etc. ahead.
Only the extant sections of the ["terms & wording" page](https://github.com/sailfishos-patches/patchmanager/wiki/Terms-and-wording) are O.K. from my POV.
